### ISSUE TITLE
purchased dedicated node, what can I do next

### ISSUE THREAD

**Issue by Pelekacz** [stakwork/sphinx-kotlin#278]
I purchased dedicated note downloaded the latest release on Android and now I don't know what to do next, please advise
Thank you,
Peter

**Comment by pitoi** [comment]
After your purchase, you should see a QR code. Scan that code from the welcome screen in the Sphinx app to connect the node with your app. If you don't have the QR or text string anymore, email support @ stakwork.com and we can try to recover it.

**Comment by Pelekacz** [comment]
It turned out I have purchased the Lite node thinking I was purchasing the dedicated node (are you going to have more available there is constant 0 available), in any case I would like to learn how to use it with the Lite node in the meantime. I did copy the string of characters and I clicked on the APK downloader, but it only showed me the newest release and one of the links allowed me to download zip files which I extracted but nothing else happened I don't know where should I place the string of characters which I copied from the node?

**Comment by pitoi** [comment]
If you email our support address above we can get you a dedicated node, we make some available but they go fast

**Comment by Pelekacz** [comment]
Yes that would be great, could you make a dedicated node for me available?
Thank you,
Peter

**Comment by Pelekacz** [comment]
Hello,
Could you make one dedicated node available for me for purchase?
Thank you very much,
Peter Kaczmarzyk

**Comment by Pelekacz** [comment]
Hi Paul,
I did email your stuff for the dedicated node,
Thank you,
Peter
